anoncreds/src/services/verifier.rs

```diff
@@ -30,10 +30,10 @@ static INTERNAL_TAG_MATCHER: Lazy<Regex> =
 pub fn verify_presentation(
     presentation: &Presentation,
     pres_req: &PresentationRequest,
-    schemas: &HashMap<SchemaId, &Schema>,
-    cred_defs: &HashMap<CredentialDefinitionId, &CredentialDefinition>,
-    rev_reg_defs: Option<&HashMap<RevocationRegistryId, &RevocationRegistryDefinition>>,
-    rev_regs: Option<&HashMap<RevocationRegistryId, HashMap<u64, &RevocationRegistry>>>,
+    schemas: &HashMap<String, &Schema>,
```

Comment:
Like here @blu3beri, where we have a map of schem_id => schema

Reply:
Yeah internally this works very well and is the best solution.